spec/inputs/jdbc_spec.rb

```diff
@@ -164,7 +163,7 @@
       plugin = LogStash::Inputs::Jdbc.new(mixin_settings.merge(settings))
       plugin.register
 
-      insist { File.exists?(settings["last_run_metadata_path"]) }
+      expect(File).not_to(be_exists(settings["last_run_metadata_path"]))
```

Comment:
avoid writing so many parentheses as in here, it makes reading this like complicated. Use something like in https://www.relishapp.com/rspec/rspec-expectations/v/3-3/docs/built-in-matchers/exist-matcher